SociallyUndead/Eventhandler.lua

```diff
@@ -10,6 +10,7 @@ local playerDurabilities = {}
 local itemTrackingTable = {}
 local similarItemIds = {}
 local addonMessagePrefix = "SU"
+local lootWhisper = { toggled = false, zone = nil }

```

Comment:
this could eventually be a list of zones we write out to preferences so you dont have to retrigger it everytime just when u want to change something